readthedocs/config/config.py

```diff
@@ -146,6 +146,8 @@ class BuildConfigBase:
         'mkdocs',
         'submodules',
     ]
+    valid_build_images = [k.split(':')[1] for k in DOCKER_IMAGE_SETTINGS]
+    default_build_image = DOCKER_DEFAULT_VERSION
```

Comment:
The default build image are different from v1 and v2. v1 uses 2.0, v2 uses `latest`.

Reply:
:thinking: I think both should use `stable` as default image.

Having different image default for v1 and v2 is confusing. Also, defaulting to a fixed version like `2.0` will make us to be tied to that version. Besides, `2.0` is deprecated.

On the other hand, I think that defaulting to `latest` is not a good practice since it may be not tested enough to use it widely as default.

Reply:
We will be changing the builds for users using v1, we shouldn't change the spec.

Using latest was decided in https://github.com/rtfd/readthedocs.org/pull/4084#discussion_r192453324

Reply:
We can default to `latest` as that comment says. I don't agree, but it's not a huge problem, though.

Regarding default to `2.0` for `v1` is not a good decision to me. I'd like to change that to `stable` (this is a non-breaking change and actually is more stable). Right now, we are defaulting to ConfigV1 for new projects, and ConfigV1 is defaulting to `2.0`, so in the end, all new projects will be using a `2.0` which is a deprecated build image.

I'm -1 on defaulting to a deprecated Docker build image for new projects.

Reply:
If there isn't a technical limitation to why we can't use the same image for both, I agree that both should point to the same image.

However, I don't think that image should be `stable` by default. Everyone should be using `latest` now, or `3.0` by default. Not continue to use `2.0`. The `stable` image was meant as a way for users to opt into an older image if they are having serious problems with the newer image.

Reply:
To clarify, terminology here has swapped, as per our conversation on docker image versioning. `latest` doesn't mean bleeding edge to us, that is what RCs are. `latest` is the most recent image that we have vetted and consider production ready.

Reply:
OK.

`stable` is `3.0` at this moment. Anyways, using `latest` should not be a problem and will keep our projects up to date, but we will need to careful when releasing new images, since the change will affect the projects immediately.

Considering that we have a feature flag now to use `testing`, this will allow us to test RCs easily.

Reply:
> If there isn't a technical limitation to why we can't use the same image for both, I agree that both should point to the same image.

I think there isn't a technical limitation, but I'm not 100% sure yet.

I will default both to `latest` an test locally how it behaves as a first step.